### DISCUSSION TITLE
Karavan JMS component-Jndi connect

### DISCUSSION THREAD

**Discussion by skumaravelan** [apache/camel-karavan#1375] [Q&A] [answered]
### Ask a question

How can connect to Solace topic using Camel
 JMS component, could any one share DSL for the working version using Karavan ?

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by davsclaus** [comment]
look for how camel-jms component (and spring jms) would configure to Solace. You need to have their JMS client and how to setup all of that. 

Then configure a JMS ConnectionFactory for this that camel-jms uses (such as via beans or properties files) and you can use that in karavan

**Comment by skumaravelan** [comment]
@davsclaus You mean in the camel repo . I tried to find example 
for solace with connection factory and jndi  property file. I am not able find a good example.

**Reply by davsclaus** [reply, marked as answer]
Look for solace with spring jms and jndi property file - camel uses spring jms and it would be same/similar configuration
